packages/file-type-icons/src/FileIconType.ts

```diff
@@ -11,7 +11,8 @@ export enum FileIconType {
   genericFile = 3,
   listItem = 4,
   sharedFolder = 5,
-  multiple = 6
+  multiple = 6,
+  stream=7
```

Comment:
Minor formatting issue. This should have been corrected automatically by Prettier when you committed, which makes me think your local copy isn't quite set up properly. 
```suggestion
  stream = 7
```